packages/storybook/stories/ControlBar/Buttons/Participants/snippets/WithCustomRender.snippet.tsx

```diff
@@ -73,11 +74,12 @@ export const ParticipantsButtonWithCustomRenderExample: () => JSX.Element = () =
 
   const customOnRenderText = (): JSX.Element => {
     return (
-      <Label key={'participantsCustomLabelKey'} style={{ color: 'darkviolet', fontStyle: 'italic' }}>
+      <Label key={'participantsCustomLabelKey'} style={{ color: 'darkviolet', fontStyle: 'italic', padding: '0rem' }}>
         who is on this call
       </Label>
     );
   };
+
   return (
     <FluentThemeProvider>
       <ParticipantsButton
```

Comment:
Can we also remove this scroll bar? (not a regression in this PR)
![image](https://user-images.githubusercontent.com/2684369/156097957-b8e64de5-f8b6-44a2-b676-010242a9b413.png)